pennylane_lightning/src/gates/Constant.hpp

```diff
@@ -200,31 +200,30 @@ namespace Pennylane::Gates::Constant {
  * @brief Number of wires for generators besides multi-qubit gates
  */
 [[maybe_unused]] constexpr std::array generator_wires = {
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::PhaseShift,
-                                               1},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::RX, 1},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::RY, 1},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::RZ, 1},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::IsingXX, 2},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::IsingXY, 2},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::IsingYY, 2},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::IsingZZ, 2},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::CRX, 2},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::CRY, 2},
-    std::pair<GeneratorOperation, std::size_t>{GeneratorOperation::CRZ, 2},
-    std::pair<GeneratorOperation, std::size_t>{
-        GeneratorOperation::SingleExcitation, 2},
-    std::pair<GeneratorOperation, std::size_t>{
+    std::pair<GeneratorOperation, size_t>{GeneratorOperation::PhaseShift, 1},
```

Comment:
why are we changing `std::size_t` to `size_t`? 
Should we have a change (or addition) of headers since the size_t is C-like?

Reply:
Will just revert, as discussed [here](https://github.com/PennyLaneAI/Lightning-Unification/pull/25#discussion_r1287354203).